scripts/generate_index_pages.ts

```diff
@@ -151,37 +144,31 @@ async function main() {
     );
   }
 
-  for (const stateCode in STATES) {
-    const stateName = (STATES as any)[stateCode];
-    const relativeSiteUrl = `/us/${stateCode.toLowerCase()}/`;
-    const relativeImageUrl = `/states/${stateCode.toLowerCase()}`;
-    await buildLocationPages(
-      builder,
-      relativeSiteUrl,
-      relativeImageUrl,
-      stateName,
-    );
-  }
-
-  for (const fips of COUNTIES) {
-    if (BLACKLISTED_COUNTIES.includes(fips)) {
-      continue;
-    }
-    const county = findCountyByFips(fips);
-    assert(county, 'Failed to find county ' + fips);
-    const stateCode = county.state_code;
-    const locationName = `${county.county}, ${(STATES as any)[stateCode]}`;
-    const relativeSiteUrl = `/us/${stateCode.toLowerCase()}/county/${
-      county.county_url_name
-    }`;
-    const relativeImageUrl = `counties/${fips}`;
+  regions.states.forEach(async (state: State) => {
+    const stateCode = state.stateCode.toLowerCase();
+    const relativeImageUrl = `/states/${state.stateCode.toLowerCase()}`;
+    const relativeSiteUrl = `/us/${stateCode}/`;
     await buildLocationPages(
       builder,
       relativeSiteUrl,
       relativeImageUrl,
-      locationName,
+      state.fullName,
     );
-  }
+  });
+
+  regions.counties
+    .filter(hasLocationSummary)
+    .forEach(async (county: County) => {
+      const stateCode = county.stateCode.toLowerCase();
+      const relativeSiteUrl = `/us/${stateCode}/county/${county.urlSegment}`;
+      const relativeImageUrl = `counties/${county.fipsCode}`;
```

Comment:
This might not be specific to this, but why is the relative image url for county `counties/...` but the relative image url for states is `/states/...`? I guess it's using url join so shouldn't matter

Reply:
It doesn't matter, we use `urlJoin` to join `relativeImageUrl` with the rest :)